src/components/tooltip/README.md

```diff
@@ -127,7 +129,7 @@ By default, the tooltip (`.bx--tooltip`) goes right under `<body>`. You can chan
 
 ### Definition tooltip
 
-This tooltip variation does not use any JavaScript and should be used to define a word. For anything more advanced please use the main variation.
+This tooltip variation is for regular use case of tooltip, e.g. giving the user more text information about something, like defining a word. This works better than interactive tooltip in regular use cases because info icon used in interactive tooltip gets really busy when it’s used several times on a page. This tooltip variation does not use any JavaScript. If there are actions a user can take in the tooltip (e.g. a link or a button), use the main variation.
```

Comment:
I'd perhaps reword the second line to `This works better than interactive tooltip in regular use cases because the info icon used in the interactive tooltip can be very repetitive when it’s shown several times on a page`